nolibc/include/sys/stat.h

```diff
@@ -11,7 +11,10 @@ struct stat {
 #define S_IFMT 0
 #define S_IFREG 0
 #define S_ISREG(x) (0)
```

Comment:
```suggestion
#define S_ISREG(x) 0
```
Nitpick, the parentheses were not useful here.